Update README.adoc (#129)
jakartaee/data: README.adoc

== Introduction

> The Jakarta Data specification provides an API for easier data access to a variety of database types, both relational and NoSQL. A Java developer can access those repositories a number of ways, including the ability to compose custom query methods on a Repository interface.

Reviewer: ```suggestions
The Jakarta Data specification provides an API for easier data access to various database types, both relational and NoSQL. A Java developer can access those repositories in several ways, including composing custom query methods on a Repository interface.
```

Author: good point.
